cli/src/commands/new.rs

```diff
@@ -23,6 +23,8 @@ impl NewCmd {
         // Clone the repository.
         let output = Command::new("git")
             .arg("clone")
+            .arg("-b")
```

Comment:
In general, in script you should pick long options, like `--branch` in this case.  That helps with readability, and if you make a typo, it's more likely that you get a proper error message, instead of silently doing something else entirely.